test/kitchen/drivers/ec2-driver.yml

```diff
@@ -37,10 +37,12 @@ driver:
   <% else %>
   security_group_ids: [ "sg-7fedd80a","sg-46506837" ]
   <% end %>
+  <% if ENV['KITCHEN_OSVERS'] != "ubuntu-16-04-4.4" %>
```

Comment:
This is fine for now, but won't work if any kitchen test in the future specifies this platform + other ones (like `KITCHEN_OSVERS: ubuntu-16-04-4.4,ubuntu-18-04`).

I think this is fine because it's an edge case, only one test suite uses this platform, and defines a test with only that platform, but that behavior should at least be mentioned in a comment.